src/SFA.DAS.ApplyService.Web/Services/RoatpTaskListWorkflowService.cs

```diff
@@ -90,6 +109,25 @@ private  int SectionHasCompletedQuestions(ApplicationSection section)
             return answeredQuestions;
         }
 
+        private string GetSectionText(int completedCount, ApplicationSection section,  bool sequential)
```

Comment:
GetSectionText and SectionText appear to be doing pretty much the same thing?

Reply:
They do, but the 'SectionCompleted' is being got from the database and used in the SectionText method, which is still needed for section 1 until we get it to conform.  The follow up bugfix story will remove one of them